pkg/cloud/services/autoscaling/autoscalinggroup.go

```diff
@@ -284,35 +284,35 @@ func (s *Service) DeleteASG(name string) error {
 }
 
 // UpdateASG will update the ASG of a service.
-func (s *Service) UpdateASG(scope *scope.MachinePoolScope) error {
-	subnetIDs, err := s.SubnetIDs(scope)
+func (s *Service) UpdateASG(machinePoolScope *scope.MachinePoolScope) error {
```

Comment:
the renaming of the variable from `scope` to `machinePoolScope` is necessary to be able to access the package `scope` and the function `scope.ReplicasExternallyManaged`